src/engine/xrinput.js

```diff
@@ -6,6 +6,8 @@ class XRInputClass
 {
     constructor()
     {
+        this.showDebugOutput = true;
```

Comment:
It might be nice to have this as a configurable setting

Reply:
Agreed. PongXR has a major refactor of all debugs (physics, input, networking) to a single debug flag: State.debugMode, which defaults to true but we should discuss further QoL enhancements (CLI arg, default to true in dev, false in build?)